.github/workflows/build_playground_frontend.yml

```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build And Deploy Playground Frontend Application
+
+on:
+  push:
+    branches: ['master', 'release-*']
+    tags: 'v*'
+  pull_request:
+    branches: ['master', 'release-*']
+    tags: 'v*'
+    paths: ['playground/backend/**']
+  workflow_dispatch:
+
+jobs:
+  build_playground_frontend_docker_image:
+    name: Build Playground Frontend App
+    runs-on: ubuntu-latest
+    env:
+      GO_VERSION: 1.17.0
+      BEAM_VERSION: 2.33.0
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: joschi/setup-jdk@v2
+        with:
+            java-version: '8'
+      - name: install flutter
+        uses: subosito/flutter-action@v1
+        with:
+            channel: 'stable'
+      - name: Prepare Flutter lint
+        run: "flutter pub add flutter_lints --dev"
+        working-directory: playground/frontend/        
+        continue-on-error: true
+      - name: Run Lint
+        run: "flutter analyze"
+        working-directory: playground/frontend/
+        continue-on-error: true
+      - name: Run Tests
+        run: flutter test
+        working-directory: playground/frontend/
+        continue-on-error: true
+      - name: install npm
+        uses: actions/setup-node@v2
+        with:
+            node-version: '14'
+      - name: install docker linter
+        run: npm install  -g dockerlint
+      - name: lint dockerfile
+        run: dockerlint Dockerfile
+        working-directory: playground/frontend
+      - name: Setup GCP account
+        run: echo ${{ secrets.GCP_ACCESS_KEY }} | base64 -d > /tmp/gcp_access.json
```

Comment:
Where are the secrets/passwords coming from? Are they going to be visible in the GH action logs?

Reply:
Secrets comming from secrets section from repository settings, and output of values of secrets would be hidden

Reply:
Or you can show it once we merge the PR.

Reply:
if the pr needs to be merged before you can trigger the job.

Reply:
we have some github secrets implemented for some of our other github action workflows. I just don't know how they got set up